packages/block-editor/src/components/rich-text/index.js

```diff
@@ -824,51 +820,44 @@ export class RichText extends Component {
 		} );
 	}
 
-	/**
-	 * Splits the content at the location of the selection.
-	 *
-	 * Replaces the content of the editor inside this element with the contents
-	 * before the selection. Sends the elements after the selection to the `onSplit`
-	 * handler.
-	 *
-	 * @param {Array}  blocks  The blocks to add after the split point.
-	 * @param {Object} context The context for splitting.
-	 */
-	splitContent( blocks = [], context = {} ) {
-		if ( ! this.onSplit ) {
+	onSplit( record, pastedBlocks = [] ) {
```

Comment:
I think it would be good to clarify the different use cases this function is called (sorry If it's somewhere else and I missed it), something like:

 - putting the caret in the middle of a RichText and pasting
 - click "Enter" in the middle of the text
 - other use-cases I'm not aware of?

Reply:
Done in 199a1e9.